### PR TITLE
move Translator to common module

### PR DESCRIPTION
**Pull request by AndreasEhret**
moved static config part closer to config package for now. Not perfect but breaks the dependency of Translator to DeploymentConfiguration

### CODE REVIEW

File: core/src/main/java/net/sourceforge/jnlp/services/XBasicService.java [hunk 1]
```diff
@@ -261,13 +249,13 @@ private boolean exec(String cmd, String url) {
...
             OutputController.getLogger().log(OutputController.Level.MESSAGE_ALL, e.toString());
-            OutputController.getLogger().log(OutputController.Level.MESSAGE_ALL, Translator.VVPossibleBrowserValues());
+            OutputController.getLogger().log(OutputController.Level.MESSAGE_ALL, DeploymentConfiguration.VVPossibleBrowserValues());
             return false;
         }
     }
 
     private String promptForCommand(final String targetUrl, boolean aa) throws IOException {
-        String message = Translator.VVPossibleBrowserValues();
+        String message = DeploymentConfiguration.VVPossibleBrowserValues();
```
Comment (hendrikebbers) [inline core/src/main/java/net/sourceforge/jnlp/services/XBasicService.java]: add `final` pls ;)

File: core/src/main/java/net/sourceforge/jnlp/services/XBasicService.java [hunk 2]
```diff
@@ -317,7 +305,7 @@ public void arrange(String url, boolean ask) {
             JTextField urlField = new JTextField(url);
             urlField.setEditable(false);
             top.add(urlField);
-            JTextArea ta = new JTextArea(Translator.VVPossibleBrowserValues());
+            JTextArea ta = new JTextArea(DeploymentConfiguration.VVPossibleBrowserValues());
```
Comment (hendrikebbers) [inline core/src/main/java/net/sourceforge/jnlp/services/XBasicService.java]: add `final` pls ;)

File: core/src/main/java/net/sourceforge/jnlp/config/DeploymentConfiguration.java [hunk 3]
```diff
@@ -957,4 +957,18 @@ private static String loadComments(URL path) {
...
         return r.toString().trim();
     }
+
+    /**
+     * convenient method to show VVPossibleBrowserValues with all four params
+     *
+     * @return translation of VVPossibleBrowserValues with all params in
+     */
+    public static String VVPossibleBrowserValues() {
+        return R("VVPossibleBrowserValues", DeploymentConfiguration.LEGACY_WIN32_URL__HANDLER,
```
Comment (hendrikebbers) [inline core/src/main/java/net/sourceforge/jnlp/config/DeploymentConfiguration.java]: const for "VVPossibleBrowserValues"

File: core/src/main/java/net/sourceforge/jnlp/services/XBasicService.java (hunk 1 above)
Comment (AndreasEhret) [reply core/src/main/java/net/sourceforge/jnlp/services/XBasicService.java]: done

File: core/src/main/java/net/sourceforge/jnlp/services/XBasicService.java (hunk 2 above)
Comment (AndreasEhret) [reply core/src/main/java/net/sourceforge/jnlp/services/XBasicService.java]: done

File: core/src/main/java/net/sourceforge/jnlp/config/DeploymentConfiguration.java (hunk 3 above)
Comment (AndreasEhret) [reply core/src/main/java/net/sourceforge/jnlp/config/DeploymentConfiguration.java]: done

**Pull request merged**